### PR TITLE
docs: daemon.json in Server Version Docker: 18.09.0  > * (Fedora 27) Not required.

### PR DESCRIPTION
**Pull request by GloriaPG**
Follow the guide steps for Fedora, I could not run Docker with Kata Engine.

Throws me this error:

```bash
Redirecting to /bin/systemctl status  -l docker.service
● docker.service - Docker Application Container Engine
   Loaded: loaded (/usr/lib/systemd/system/docker.service; disabled; vendor preset: disabled)
  Drop-In: /etc/systemd/system/docker.service.d
           └─kata-containers.conf
   Active: failed (Result: exit-code) since Wed 2019-09-18 15:39:47 CDT; 1min 8s ago
     Docs: https://docs.docker.com
  Process: 16297 ExecStart=/usr/bin/dockerd -D --add-runtime kata-runtime=/usr/bin/kata-runtime --default-runtime=kata-runtime (code=exited, status=1/FAILURE)
 Main PID: 16297 (code=exited, status=1/FAILURE)

Sep 18 15:39:45 zatory systemd[1]: Failed to start Docker Application Container Engine.
Sep 18 15:39:47 zatory systemd[1]: docker.service: Service hold-off time over, scheduling restart.
Sep 18 15:39:47 zatory systemd[1]: docker.service: Scheduled restart job, restart counter is at 3.
Sep 18 15:39:47 zatory systemd[1]: Stopped Docker Application Container Engine.
Sep 18 15:39:47 zatory systemd[1]: docker.service: Start request repeated too quickly.
Sep 18 15:39:47 zatory systemd[1]: docker.service: Failed with result 'exit-code'.
Sep 18 15:39:47 zatory systemd[1]: Failed to start Docker Application Container Engine.
Sep 18 15:40:29 zatory systemd[1]: docker.service: Start request repeated too quickly.
Sep 18 15:40:29 zatory systemd[1]: docker.service: Failed with result 'exit-code'.
Sep 18 15:40:29 zatory systemd[1]: Failed to start Docker Application Container Engine.
```
I think the problem is because, in the lastest Server Version Docker, daemon.json works in a different way. So, only I needed delete daemon.json, is enough configure service.

### CODE REVIEW

**Comment by caoruidong** [comment]
Thanks for raising. Could you please open an issue and squash your commits?

**Comment by GloriaPG** [comment]
> Thanks for raising. Could you please open an issue and squash your commits?

Of course. I will.

**Comment by jodh-intel** [comment]
Hi @GloriaPG - I'm a little confused by this. Could you explain how you installed Kata? If you followed the automatic installation method (https://github.com/kata-containers/documentation/tree/master/install#automatic-installation), that would have created `/etc/systemd/system/docker.service.d/kata-containers.conf` automatically ("option 1"). However, if you then create `/etc/docker/daemon.json` *as well*, docker will get confused.

In summary, you need *either* create `/etc/systemd/system/docker.service.d/kata-containers.conf`  *or* create `/etc/docker/daemon.json`.

**Comment by GloriaPG** [comment]
Hi @jodh-intel could be the reason was that I follow these steps to install Kata : https://github.com/kata-containers/documentation/blob/master/install/docker/fedora-docker-install.md in manually way, I will try to use automatic-installation for testing.

**Comment by jodh-intel** [comment]
Thanks @GloriaPG.

**Comment by cmaf** [comment]
Hi @GloriaPG, were you test & resolve this with automatic install?

**Comment by cmaf** [comment]
Ping @GloriaPG?

**Comment by cmaf** [comment]
Closing for now, @GloriaPG feel free to reopen
